src/bunnicula/component/connection.clj

```diff
@@ -37,16 +43,19 @@
 
 
 (defn extract-server-config
-  [{:keys [url host port username password vhost connection-name]}]
+  [{:keys [url host port username password vhost connection-name secure?]}]
   {:post [#(string? (:host %))
           #(string? (:port %))
           #(string? (:username %))
           #(string? (:password %))
           #(string? (:vhost %))]}
   (if-let [^URI uri (and url (java.net.URI. url))]
-    (let [[username password] (string/split (.getUserInfo uri) #":")]
+    (let [[username password] (string/split (.getUserInfo uri) #":")
+          secure? (string/starts-with? url "amqps")
+          ]
```

Comment:
indent

Reply:
I'm going to run `cljstyle` against master after this is merged 👍